tests/test_casefold_migration.py

```diff
@@ -196,7 +196,7 @@ def test_migration_email(self):
 
     def test_local_db_migration(self):
         with patch("sydent.util.emailutils.smtplib") as smtplib:
-            update_local_associations(self.sydent, self.sydent.db, "apply")
+            update_local_associations(self.sydent, self.sydent.db, True, False)
```

Comment:
very minor
```suggestion
            update_local_associations(self.sydent, self.sydent.db, send_email=True, dry_run=False)
```

This will just help the readability of the code, especially for people coming in fresh, especially when we have multiple boolean flags (it can be hard to keep the order in your head).

& similar for below